### PR TITLE
Purge v1alpha1 routing from e2e

### PR DESCRIPTION
**Pull request by ijsnellf**
Cleans up the e2e code.

### CODE REVIEW

File: tests/e2e/tests/pilot/ingress/main_test.go [hunk 1]
```diff
@@ -1,53 +0,0 @@
...
-}
-
-func TestMain(m *testing.M) {
-	flag.Parse()
-	//cc, err := framework.NewCommonConfig("pilot_ingress_test")
-	//if err != nil {
-	//	os.Exit(-1)
-	//}
-	//tc.CommonConfig = cc
-}
```
Comment (ijsnellf) [inline tests/e2e/tests/pilot/ingress/main_test.go]: this was disabled, and only meant for v1alpha1 routing

File: tests/e2e/tests/pilot/ingress/controller_test.go [hunk 2]
```diff
@@ -1,340 +0,0 @@
-// Copyright 2017 Istio Authors
```
Comment (costinm) [inline tests/e2e/tests/pilot/ingress/controller_test.go]: I think we still need this, ingress should still be supported.

**Comment by costinm** [comment]
Need to confirm the status of ingress and the ingress test, rest seems good.

File: tests/e2e/tests/pilot/ingress/controller_test.go (hunk 2 above)
Comment (ijsnellf) [reply tests/e2e/tests/pilot/ingress/controller_test.go]: maybe just create an issue to add tests for ingress with the v2 implementation?

**Pull request merged**